### ISSUE TITLE
crash in case regenerating spritesheet

### ISSUE THREAD

**Issue by XenoRoot** [estliberitas/node-thumbnails-webvtt#1]
In spritesheet.js, there is a confused variable called 'sprite'

One is

```
6   sprite = require('node-sprite')
108 sprite.sprite(sheetName, spriteOptions, function(err, globalSprite) {
```

The other is

```
121 sprite = lodash.find(sprites, {filename: filename})
```

One of the two should be renamed.

Thanks for your contribution.

**Comment by estliberitas** [comment]
Hah, really, you're right. Will fix soon.

**Comment by estliberitas** [comment]
Have no time today, will do tmrw.

**Comment by estliberitas** [comment]
Fixed and published `thumbnails-webvtt@0.0.4`:

```
npm install thumbnails-webvtt@0.0.4 --save
```
